spec-0014/index.md

```diff
@@ -0,0 +1,410 @@
+
+###### Narrative documentation and tutorials (long-form content)
+
+For longer-form narrative content such as tutorials, how-to guides, and explanatory documentation:
+
+- `jupyterlite-sphinx` offers directives to render long-form content written in notebooks (the [`jupyterlite` directive](https://jupyterlite-sphinx.readthedocs.io/en/stable/directives/jupyterlite.html), the [`notebooklite` directive](https://jupyterlite-sphinx.readthedocs.io/en/stable/directives/notebooklite.html), and so on), which can be configured to open notebooks in new tabs or iframes within the same tab. It supports both traditional IPyNB files and MyST Markdown notebooks. Unlike the `try_examples` directive, these directives do not work with reST-based content. We recommend using MyST Markdown notebooks, as they integrate well with version control.
+- The Sphinx-Gallery project offers a [JupyterLite integration, built with `jupyterlite-sphinx`](https://sphinx-gallery.github.io/stable/configuration.html#jupyterlite) which enables a "JupyterLite" button in the secondary sidebar to open a notebook in the JupyterLite deployments. It provides a `notebook_modification_function` configuration option via `conf.py` that can be used to configure code cells or Markdown cells at the top of a notebook, with custom `%pip`-install commands, imports from other libraries, or extra admonitions.
+
+---
+
+Once interactive documentation has been configured and deployed, the ["Updating package and kernel versions for Pyodide-powered or emscripten-forge-powered websites"](#Updating-package-and-kernel-versions-for-Pyodide-powered-or-emscripten-forge-powered-websites) section contains information on how to increment the packages' and distribution/kernel's versions.
+
+An end-to-end example of the above steps is also available; see [the `jupyterlite-sphinx-demo` project](#an-end-to-end-example).
+
+#### What distribution should projects choose: Pyodide or emscripten-forge?
+
+<!-- I have not had a better answer to this beyond "it depends" so I don't know how to end this section even though I have known to start it -->
+
+There are a few differences between the two distributions in terms of runtimes and packaging, and the kernels, which are worth noting:
+
+- Available runtimes: It is possible to use the emscripten-forge distribution only through the JupyterLite project, unlike Pyodide, which also supports Node.js and native browser-based JavaScript runtimes.
```

Comment:
I don't think I understand this sentence. Does it mean that 
- Pyodide needs extra stuff to work, unlike emscripten-forge which works by itself with jupyterlite; 
or
- emscripten-forge only supports jupyterlite, unlike Pyodide which also supports node.js and native browser-based JS runtimes
?

Reply:
Yes, it means the latter, as it's a limitation on emscripten-forge's side and Pyodide's list of supported runtimes is larger.

However, now that I think of it, for a reader who is coming with no information about the topic and only wants to know more about and implement interactive documentation, I suppose the difference does not matter to them? The deployments (at least the ones based on this SPEC) will use JupyterLite anyway and not anything else, so perhaps we can drop this point?

Reply:
Yes I think that makes sense. For that kind of inside knowledge they can definetely refer to the pyodide/emscripten-forge docs.